eng/Directory.Build.Common.targets

```diff
@@ -98,6 +98,15 @@
 
   </ItemGroup>
 
+  <!-- Remove packages built into the .NET 6+ runtime -->
+  <ItemGroup Condition="'$(TargetFramework)' == 'net6.0' OR '$(TargetFramework)' == 'net8.0'">
+    <PackageReference Remove="System.Buffers" />
+    <PackageReference Remove="System.Net.Http" />
+    <PackageReference Remove="System.Threading.Channels" />
+    <PackageReference Remove="System.Text.Encodings.Web" />
+    <PackageReference Remove="System.Text.Json" />
```

Comment:
Can you share more about why these are needed?

Reply:
This trims polyfill packages that we're including as direct references for the `netstandard2.0` case but which are built into modern frameworks.   This was also part of the Core meeting discussion from ~9 months back.

https://gist.github.com/jsquire/d7a98cb42b9eaa9e3b942a7acd33a233